runtime/lua/vim/cmd_preview.lua

```diff
@@ -0,0 +1,35 @@
+local M = {}
+
+function M.open_win(opts)
```

Comment:
Bikeshed: the `_win` suffix is redundant

```suggestion
function M.open(opts)
```

Reply:
Is it though, opening preview isn't the same thing as opening the preview window, since you can have preview without a window with `inccommand=nosplit`